script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt

```diff
@@ -49,7 +49,7 @@ I didn't think that Arcueid――someone who takes battle seriously――would l
 <Page15>
 "I told you, it's fine.//ruby dots <そ|・><れ|・>
 Either we keep running for three days, or we find and kill Vlov tonight.
-They both have the same chance of success either way. The only difference is your own chances of survival."
+They both have the same chance of success either way. The only difference is your own chance of survival."
```

Comment:
I like "chances" more but maybe I'm trippin

Reply:
chance is better imo